### DISCUSSION TITLE
address fields are black with black descriptors in dark page setting...state, city, zip, etc cannot be seen

### DISCUSSION THREAD

**Discussion by KK1L** [woocommerce/woocommerce#45470] [Checkout Flow]
address fields are black with black descriptors in dark page setting...state, city, zip, etc cannot be seen

![Capture](https://github.com/woocommerce/woocommerce/assets/73606166/5d51d392-d5e6-4824-b2f6-142245098bd9)

**Comment by senadir** [comment]
Hey! Checkout supports a dark mode that you can enable from the editor

<img width="287" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/567e74a4-dc77-4f10-8c1e-7c6bb9bc6d2e">

<img width="729" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/23b56425-637a-47a9-bc86-e798be942b3e">
